### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/zeek-docs/commit/26928405db292c1eaa97baa23eab980b7463ec48#commitcomment-90296894

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/zeek-docs/commit/8ba614526b7a31cc35add69148d51073dec102c3

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -37,7 +37,7 @@ Doxygen-like feature, named "Zeekygen".  The bulk of auto-generated content
 lives under the ``scripts/`` directory or has a file name starting with
 "autogenerated", so if you find yourself wanting to change those, you should
 actually look at at doing those changes within the `Zeek repo`_ itself rather
-than here, so see the next section for how Zeeygen docs can be (re)generated.
+than here, so see the next section for how Zeekygen docs can be (re)generated.
```
Comment (jsoref) [inline README:40]: This is by far the most interesting/worrisome change

File: cluster-setup.rst [hunk 2]
```diff
@@ -318,7 +318,7 @@ Linux
 
 While netmap isn't included in the Linux kernel, it can be installed as
 an out-of-tree kernel module.
-See the project's `Github repository <https://github.com/luigirizzo/netmap>`_
+See the project's `GitHub repository <https://github.com/luigirizzo/netmap>`_
```
Comment (jsoref) [inline cluster-setup.rst:321]: brand

**Review by jsoref — COMMENTED** [review]
Some corrections suggested by Google Sheets.

**Review by timwoj — CHANGES_REQUESTED** [review]

File: logs/smb.rst [hunk 3]
```diff
@@ -1199,25 +1199,25 @@ layer of the decode we want to see.
                   .... .... .... .... .... ..0. .... .... = Tenth: Tenth is NOT SET
                   .... .... .... .... .... .0.. .... .... = Eleventh: Eleventh is NOT SET
                   .... .... .... .... .... 0... .... .... = Twelfth: Twelfth is NOT SET
-                  .... .... .... .... ...0 .... .... .... = Thitteenth: Thitteenth is NOT SET
+                  .... .... .... .... ...0 .... .... .... = Thirteenth: Thirteenth is NOT SET
```
[the remark this answers is not in the document]
Comment (ckreibich) [reply logs/smb.rst:1202]: Agreed.

[the remark this answers is not in the document]
Comment (jsoref) [reply logs/smb.rst:1202]: Ah. Indeed, https://github.com/wireshark/wireshark/blob/f3a96bc18cfc1b0b434682fccc313c477d2ca86e/epan/dissectors/packet-dcerpc-atsvc.c#L161
Ok, added to my shopping list.

**Review by ckreibich — APPROVED** [review]
Thanks!

**Pull request merged**